master/buildbot/reporters/utils.py

```diff
@@ -87,6 +87,32 @@ def getDetailsForBuild(master, build, wantProperties=False, wantSteps=False,
     return ret
 
 
+@defer.inlineCallbacks
+def getDetailsForBuildRequest(master, buildrequest):
```

Comment:
Let's use snake_case for new code. Eventually all of buildbot will use snake_case like normal Python code.